media-translation/cloud-client/translate_from_file.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud Media Translation sample application.
+
+Example usage:
+    python translate_from_file.py resources/audio.raw
+"""
+
+# [START media_translation_translate_from_file]
+from google.cloud import mediatranslation
+
+
+def translate_from_file(file_path='path/to/your/file'):
+
+    client = mediatranslation.SpeechTranslationServiceClient()
+
+    speech_config = mediatranslation.TranslateSpeechConfig(
```

Comment:
Nit: so up to you, personally I like to have the variables names match the argument they'll be used in later, but the object name is `speech_config` 

```suggestion
    audio_config = mediatranslation.TranslateSpeechConfig(
```

Same thing for `config` below --> `streaming_config`, but again object name isn't the same.